data-raw/standard-format-data-prep/rst_catch_standard_format.Rmd

```diff
@@ -400,28 +401,36 @@ butte_trap_pre_2015 <- butte_rst %>%
   group_by(wy) %>% 
   summarize(min = min(date))
 
-butte_trap_camp_dates <-  butte_trap_camp |>
+butte_trap_camp_dates <-  butte_trap_camp %>%
   # no fish are processed on these days so don't want to include catch data
-  filter(fishProcessed != "N/A; not a sampling visit") |> 
+  filter(fishProcessed != "N/A; not a sampling visit") %>% 
   mutate(wy = ifelse(month(visitTime) %in% 10:12, year(visitTime) + 1, year(visitTime))) %>% 
   group_by(wy) %>% 
   summarize(min = min(visitTime))
 
-butte_trap_combined <- bind_rows(butte_trap_pre_2015, butte_trap_camp_dates) |>  rename(trap_min = min)
+butte_trap_combined <- bind_rows(butte_trap_pre_2015, butte_trap_camp_dates) %>%  rename(trap_min = min)
 
 # min date in catch and trap aligns 
 butte_min_date_compare <- full_join(butte_rst_min, butte_trap_combined)
 ```
 
 
 ```{r, final_butte_edits}
+# find all dates that a trap that is not okie dam fyke trap is running
```

Comment:
I like the way you did the filtering here. This assumes that the NA subsites are not fyke sites which I think is the right call. I debated about adding a subsite label to the NA subsites but ended up deciding that it is more clear to leave as NA.